Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -409,8 +409,12 @@ protected Subscription CreateSubscription(Universe universe, Security security,
                     var enqueable = new EnqueueableEnumerator<BaseData>();
                     _customExchange.SetDataHandler(config.Symbol, data =>
                     {
-                        enqueable.Enqueue(data);
-                        if (subscription != null) subscription.RealtimePrice = data.Value;
+                        // Only enqueue new data if there is none (LastEnqueued == null) or if newer 
+                        if (enqueable.LastEnqueued == null || data.EndTime > enqueable.LastEnqueued.EndTime)
+                        {
```

Comment:
We put that `Log.Trace` after that [line](https://github.com/AlexCatarino/Lean/blob/quandl-issue373/Engine/DataFeeds/Enumerators/FastForwardEnumerator.cs#L71), because that if-condition was never met since `_current` had been always `null`. If `_current` on that line is always `null`, there is no other place other then this point we can check for new data.
I can run this branch in my machine for one or two days straight to confirm what I've observed so far.